src/text-editor.coffee

```diff
@@ -285,6 +285,10 @@ class TextEditor extends Model
             @preferredLineLength = value
             displayLayerParams.softWrapColumn = @getSoftWrapColumn()
 
+        when 'maxScreenLineLength'
+          if value isnt @maxScreenLineLength
+            @maxScreenLineLength = value
```

Comment:
Since the `maxScreenLineLength` field affects value returned by `getSoftWrapColumn`, I think we should include the following statement here:

```coffee
displayLayerParams.softWrapColumn = @getSoftWrapColumn()
```

This way, if you *just* update the `maxScreenLineLength` field, we'll still recompute the text layout in the display layer.